pkgs/by-name/li/libmsquic/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lttng-tools,
+  libatomic_ops,
+  perl,
+  coreutils,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libmsquic";
+  version = "2.4.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "msquic";
+    rev = "v${finalAttrs.version}";
+    sha256 = "WveyZ9rMevLTc5C4cgMFcnj0O6Hd+HcfU8ccD6VBgyU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    perl
+    coreutils
```

Comment:
I think this is redundant because of coreutils being provided in the stdenv. Remove it, give it a try, if it doesn't fail then you can push the change.

Reply:
I need access to `"${coreutils}/bin/env"` and Im not sure how I can pull that from stdenv itself. I would appreciate your input if you know what I could do.

Reply:
Ok, so then you don't need `coreutils` in `nativeBuildInputs` then.

Reply:
Done